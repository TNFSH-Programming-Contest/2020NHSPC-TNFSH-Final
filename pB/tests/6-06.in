4
-720.606 743.82
-720.606 -189.198
-973.704 -189.198
-973.704 743.82
475.827 810.643
