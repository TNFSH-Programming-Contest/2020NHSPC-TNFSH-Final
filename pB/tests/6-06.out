5230453.029
